### PR TITLE
Add workflow to automatically add issue to the ROADMAP project

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
I little poked around the Github actions to triage Github projects. And looks like [alex-page/github-project-automation-plus](https://github.com/marketplace/actions/github-project-automation) is the simplest one.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/opened-issues-triage.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Move new issues into Triage
```
Comment (nghialv) [inline .github/workflows/opened-issues-triage.yml:1]: "Move new issues into Future column"

**Comment by nakabonne** [comment]
Thank you for your review. Fixed them.

**Comment by khanhtc1202** [comment]
🏃 
/approve

**Pull request merged**